src/test/java/org/example/springbootintro/TestUtils.java

```diff
@@ -1,8 +1,11 @@
 package org.example.springbootintro;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.http.MediaType;
 
 public class TestUtils {
+    public static final MediaType JSON_CONTENT_TYPE = MediaType.APPLICATION_JSON;
```

Comment:
there is no need to make some enum a constant, if you do not like how long it looks - use static import for it